invenio_users_resources/services/schemas.py

```diff
@@ -162,6 +162,7 @@ class SystemUserSchema(BaseGhostSchema):
         dump_only=True,
     )
     username = fields.Constant(_("System"), dump_only=True)
+    email = fields.Constant(_("system@inveniosoftware.org"), dump_only=True)
```

Comment:
I don't think you need translation here...

```suggestion
    email = fields.Constant("system@inveniosoftware.org", dump_only=True)
```